src/index.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+
+var Image = require('image-js');
+
+const defaultOptions = {
+    lowThreshold: 10,
+    highThreshold: 30,
+    blur: 1.1,
+    brightness: 255
+};
+
+
+/**
+ * Find edges in an image using the Canny algorithm
+ * @param {Image} image
+ * @param {object} options
+ * @param {number} [options.lowThreshold=10] : first threshold for the hysteresis procedure.
+ * @param {number} [options.highThreshold=30] : second threshold for the hysteresis procedure.
+ * @param {number} [options.blur=1.1] : sigma parameter for gaussian filter step.
+ * @param {number} [options.brightness] : values assigned to each edge pixel on the result image.
+ * @return {Image} : an image with the edges at options.brightness value.
+ */
+function cannyEdgeDetector(image, options) {
+    options = Object.assign({}, defaultOptions, options);
```

Comment:
please use `image.checkProcessable` to only accept images that you can handle at the moment (8 bit it seems)